packages/calcite-design-tokens/README.md

```diff
@@ -0,0 +1,33 @@
+# Calcite Design Tokens
+
+## Getting Started
+
+Install the design tokens module.
+
+```bash
+npm install @esri/calcite-design-tokens
+```
+
+Reference tokens in your CSS.
```

Comment:
Can you change the period to a colon or make it a (more) complete sentence. Same note for "Install the design tokens module."